monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md

```diff
@@ -1,6 +1,6 @@
-# Portefueille Monero, démarrage rapide
-*22/04/2019*  
-Almutasim  
+# Portefeuille Monero, démarrage rapide
+*22/04/2019* Almutasim  
+*28/07/2019* mis à jour
```

Comment:
I don't thin the other documents have a note after the date. I think `mis à jour` should be removed, but better wait for @lh1008 's review

Reply:
Yes please remove `mis à jour`.